spec/models/manageiq/providers/ansible_tower/provider_spec.rb

```diff
@@ -41,4 +41,16 @@
       expect(Hardware.count).to              eq(0)
     end
   end
+
+  context "#url=" do
+    it "with full URL" do
+      subject.url = "https://server.example.com:1234/api/v1"
+      expect(subject.url).to eq("https://server.example.com:1234/api/v1")
+    end
+
+    it "missing scheme" do
+      subject.url = "server.example.com:1234/api/v1"
+      expect(subject.url).to eq("https://server.example.com:1234/api/v1")
+    end
```

Comment:
@bdunne We also need a test for when only the hostname is passed.  The user should be able to pass `server.example.com`.